blog/manifest_v3_privacy.md

```diff
@@ -0,0 +1,128 @@
+title: Manifest v3: Improving Privacy?
+subtitle: How Chrome's changes will reduce user privacy
+author: privacy team
+type: article
+publish: True
+date: 2019-06-18
+tags: blog, extensions, privacy, chrome
+header_img: blog/analytics/analytics.png
```

Comment:
Is this file already available in the repository?

Reply:
We didn't make a header image for this post yet. cc @ecnmst

Reply:
Let's use the same one the Adblocker performance Study has.